### PR TITLE
feat: added input component

### PR DESCRIPTION
**Pull request by sharmivijay**
Kindly review and add your comments

### CODE REVIEW

File: styles/variables.css [hunk 1]
```diff
@@ -16,4 +16,6 @@
     --info-border:#bee5eb;
     --info-text:#0c5460;
     --black-default:black;
+    --green-default:green;
```
Comment (bhanuprakashj) [inline styles/variables.css]: please use hex codes for black and green colors

**Review by bhanuprakashj — COMMENTED** [review]
code looks good to me !! other that a minor change suggested

File: styles/variables.css (hunk 1 above)
Comment (sharmivijay) [reply styles/variables.css]: i missed it.. I will update that

File: styles/input.css [hunk 2]
```diff
@@ -0,0 +1,47 @@
+
+
+
+
```
Comment (dev-enforced) [inline styles/input.css]: Remove these extra lines of code

File: styles/input.css [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+.error-input{
+    display: flex;
+    align-items: flex-start;
+    margin: 1px ;
+    color: var(--red-default);
+    border-color: var(--red-default);
+    border-width: 2px;
+    font-weight: bold;
+}
```
Comment (dev-enforced) [inline styles/input.css]: .error-input has similar properties as compared to that of .valid-input so you can write as 
.valid-input,.error-input on L28 
and in error-input you can just mention the color and border-color properties as mentioned

**Review by dev-enforced — APPROVED** [review]
The code looks good to me well structured one 
Do document it accordingly later on
Suggested some minor changes which you can add on later: https://deploy-preview-8--priceless-darwin-f47555.netlify.app/pages/input.html
Here it is

**Pull request merged**